app/composition/workflows/whatsapp-llm-failure-retry/workflow-factory.ts

```
@@ -0,0 +1,18 @@
+import "server-only";
+
+import { getWhatsAppWebhookDeps } from "../../whatsapp-webhook-deps";
```

Biome (2.4.4)

[error] This import is part of a cycle.

(lint/suspicious/noImportCycles)